<?xml version="1.0" encoding="UTF-8"?>

<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1024</width>
                <height>768</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Human Recognition System</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <!-- This is your main "map" or "scene" view. -->
                    <!-- You will drag and drop cameras onto this. -->
                    <widget class="QGraphicsView" name="graphicsViewScene" />
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1024</width>
                    <height>22</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>File</string>
                </property>
                <addaction name="actionExit" />
            </widget>
        </widget>
        <widget class="QStatusBar" name="statusbar" />
        <!-- This is the dockable "Camera List" on the right. -->
        <widget class="QDockWidget" name="dockWidget">
            <property name="windowTitle">
                <string>Cameras</string>
            </property>
            <attribute name="dockWidgetArea">
                <number>2</number>
            </attribute>
            <widget class="QWidget" name="dockWidgetContents">
                <layout class="QVBoxLayout" name="verticalLayout_2">
                    <item>
                        <!-- This list will show all available cameras. -->
                        <widget class="QListWidget" name="listWidgetCameras" />
                    </item>
                </layout>
            </widget>
        </widget>
        <!-- This is the toolbar at the top. -->
        <widget class="QToolBar" name="toolBar">
            <property name="windowTitle">
                <string>toolBar</string>
            </property>
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
            <addaction name="actionAdd_Camera" />
        </widget>
        <action name="actionExit">
            <property name="text">
                <string>Exit</string>
            </property>
        </action>
        <action name="actionAdd_Camera">
            <property name="text">
                <string>Add Camera</string>
            </property>
            <property name="toolTip">
                <string>Add a new camera feed</string>
            </property>
        </action>
    </widget>
    <resources />
    <connections />
</ui>